src/components/NewPhoneNumberEntry.vue

```diff
@@ -1,24 +1,40 @@
 <template>
-  <div class="flex flex-col" v-if="show">
-    <p class="text-left text-md font-semibold">
+  <div class="" v-if="show">
+    <p class="text-base mb-[10px] text-left">
       {{ label }}<span v-if="isRequired">*</span>
     </p>
     <FormKit
       type="tel"
       placeholder="xxxxxxxxxx"
-      :validation="[isRequired ? ['required'] : []]"
-      validation-visibility="dirty"
       v-model="phoneNumber"
       @keypress="isValidPhoneNumberEntry($event)"
       @input="updatePhoneNumberEntry($event)"
       ondrop="return false"
       onpaste="return false"
       :help="helpText"
+      :wrapper-class="{
+        'mx-auto': true,
+      }"
+      :inner-class="{
```

Comment:
:inner-class="{
  border: true,
  'py-2 px-2 rounded mx-auto': true,
  'border-red': this.invalid,
  'border-grey': !this.invalid,
}"